docs/examples/infer-orientation-fit-pvwatts.py

```diff
@@ -0,0 +1,88 @@
+"""
+Infer System Orientation (Azimuth & Tilt) using PVWatts
+=======================================================
+
+Infer the azimuth and tilt of a system using PVWatts data
+"""
+
+# %%
+# Identifing and/or validating  the azimuth and tilt information for a
+# system is important, as these values must be correct for future degradation
+# and system yield analysis. This example shows how to use
+# :py:func:`pvanalytics.system.infer_orientation_daily_peak` to estimate
+# a system's azimuth and tilt, using the system's known latitude-longitude
+# coordinates and an associated AC power time series.
+
+import pvanalytics
+from pvanalytics import system as sys
+import pandas as pd
+import pathlib
+import pvlib
+
+# %%
+# First, we import an AC power data stream from the SERF East site located at
+# NREL. This data set is publicly available via the PVDAQ database in the
+# DOE Open Energy Data Initiative (OEDI)
+# (https://data.openei.org/submissions/4568), under system ID 50.
+# This data is timezone-localized.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+ac_power_file = pvanalytics_dir / 'data' / \
+    'serf_east_AC_power_system_estimate.csv'
```

Comment:
Just curious, what is "estimate" referring to in this filename?

Reply:
@kanderso-nrel  I changed the naming conventions on this--I was using the original name for when I was playing with the functions, and it wasn't good. Updated in https://github.com/pvlib/pvanalytics/pull/146/commits/7ee573577bc704d95d49e2b0b8a02d5688184c47